css/main.css

```diff
@@ -0,0 +1,369 @@
+ 
+.chromeframe {
+    margin: 0.2em 0;
+    background: #ccc;
+    color: #000;
+    padding: 0.2em 0;
+}
+
+ 
+   
+p {
+    line-height: 1.33em;
+    color: #7E7E7E;
+}
+h1 {
```

Comment:
What is this needed for, it is overwriting the color of headers in index.html. Add class to DOM elements and style those by specifying the class. Cross for other tags and specifiers too.